examples/dt/uni04delta/edpm-ceph.md

```diff
@@ -0,0 +1,36 @@
+# Configuring and deploying Ceph nodes
+
+## Assumptions
+
+- The [control plane](control-plane.md) has been created and successfully
+  deployed.
+
+## Steps
+
+```bash
+# 1. Switch to openstack namespace
+
+    $ oc project openstack
+
+# 2. Change the working directory to uni04delta
+
+    $ pushd architecture/examples/dt/uni04delta
+
+# 3. Modify [ceph/values.yaml](ceph/values.yaml) file to suit your environment.
+
+    $ pushd ceph
```

Comment:
The `ceph` directory referenced on this line does not exist in the PR in the `architecture/examples/dt/uni04delta` directory. Please update these directions accordingly with the correct sub-directory so it's clear what `values.yaml` file the reader should edit.

```
[fultonj@runcible uni04delta{uniDelta}]$ find . -name values.yaml
./control-plane/nncp/values.yaml
./deployment/values.yaml
./edpm-pre-ceph/nodeset/values.yaml
./edpm-pre-ceph/values.yaml
./values.yaml
[fultonj@runcible uni04delta{uniDelta}]$ 
```

Reply:
Docs fixed.